frontend/components/mobile/MobileSearchInput.tsx

```diff
@@ -101,22 +93,20 @@ export default function MobileSearchInput(props: MobileSearchInputProps) {
         </div>
         <div className="w-full mt-14 ml-1">
           <h3 className="font-semibold text-lg">Search by Tags</h3>
-          <div className="flex flex-wrap mt-1 max-h-36 overflow-y-scroll">
+          <div className="flex flex-wrap mt-1 max-h-36 overflow-y-scroll rounded-2xl">
             <TagGroup tags={tags} size="sm" />
           </div>
         </div>
         <div className="w-full mt-10 ml-1">
           <h3 className="font-semibold text-lg">Recent Searches</h3>
```

Comment:
![image](https://github.com/user-attachments/assets/ad5679e9-456f-460a-b581-4d0696d82436)
Do we think showing recent searches in this way makes sense? I feel like the hyphen and location if there is no event name search seems a bit strange. Also tags is lowkey nonexistent how does it even show up?

Reply:
i think this can be for later when mobile serach bar gets fixed